src/main/scala/com/example/PositionOfShips.scala

```diff
@@ -28,12 +29,16 @@ class PositionOfShips(positions: Vector[Position]) {
     val totalDistance =
       pos
         .sliding(2)
-        .map { case Vector(a, b) =>
+        .map {
+        case Vector(a) => 0.0
```

Comment:
Welche änderungen meinst du?
Willst du alle Änderungen in ```PositionOfShips.scala``` in commits zerlegt haben
oder die Änderungen in ```PositionOfShips.scala``` in ein commit abgetrennt von
z.B. .gitignore änderungen?